awx/sso/tests/unit/test_fields.py

```diff
@@ -33,21 +33,23 @@ def test_internal_value_valid(self, data, expected):
 
     @pytest.mark.parametrize("data, expected", [
         ({'remove': 'blah', 'saml_attr': 'foobar'},
-            ValidationError('"blah" is not a valid boolean.')),
+            {'remove': ['Must be a valid boolean.']}),
```

Comment:
I'm commenting on the test, rather than the implementation.... but, it seems like we are losing some context here that we had before? Originally we were able to display what wasn't valid but we're moving to a more generic message?

Reply:
This one comes straight out of https://github.com/encode/django-rest-framework/pull/5881 rather than anything explicit I did.  The reasoning seems to be avoiding XSS attacks.

Reply:
So it is!